src/pages/Register/Register.js

```diff
@@ -1,7 +1,176 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Register.scss';
+import { useNavigate } from 'react-router-dom';
+
 const Register = () => {
-  return <div />;
+  // 초기값 세팅 = 아이디, 비밀번호, 비밀번호 확인, 이름, 생년원일, 휴대폰 번호
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+    pwConfirmed: '',
+    name: '',
+    birth: '',
+    phone: '',
+  });
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setPerson({ ...person, [name]: value });
+  };
+
+  // 아이디, 비밀번호 정규식
+  const passwordRegExp =
+    /^(?=.*[a-z])(?=.*\d)(?=.*[$@$!%*#?&])[a-z\d$@$!%*#?&]{8,16}$/;
+  const idRegExp = /^[a-z]+[a-z0-9]{4,12}$/;
+
+  // 통신 회원가입
+
+  const navigate = useNavigate();
+
+  const signUp = e => {
+    e.preventDefault();
+    fetch('http://10.58.5.132:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        name: person.name,
+        username: person.id,
+        password: person.pw,
+        mobile_number: person.phone,
+        birth_day: person.birth,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'SUCCESS') {
+          navigate('/');
+          alert('회원가입 성공!');
+        }
+      });
+  };
+
+  const validation =
```

Comment:
여기도 좀 더 명확하게 해주시는 게 좋을 것 같습니다. 
```suggestion
  const isAllValid =
```